src/index.html

```diff
@@ -26,25 +26,25 @@
     <!-- Open graph properties -->
     <!-- Check it here: https://developers.facebook.com/tools/debug/sharing/?q=hackupc.com -->
     <!-- Generated with: https://webcode.tools/ -->
-    <meta property="og:title" content="HackUPC 2021 | May 14&ndash;16  | Online hackathon Barcelona" />
+    <meta property="og:title" content="HackUPC 2022 | April 29&ndash;1 May  | Hybrid hackathon Barcelona" />
     <meta property="og:type" content="website" />
     <meta property="og:site_name" content="HackUPC" />
     <meta property="og:locale" content="en_GB" />
-    <meta property="og:description" content="Online hackathon in Barcelona on May 14&ndash;16 for 36 horus" />
+    <meta property="og:description" content="Hybrid hackathon in Barcelona on April 29&ndash;1 May for 36 horus" />
     <meta property="og:url" content="https://hackupc.com" />
-    <meta property="og:image" content="https://hackupc.com/ogimage.png?v=2021-v1" />
-    <meta property="og:image:secure_url" content="https://hackupc.com/ogimage.png?v=2021-v1" />
+    <meta property="og:image" content="https://hackupc.com/ogimage.png?v=2022-v1" />
+    <meta property="og:image:secure_url" content="https://hackupc.com/ogimage.png?v=-2022v1" />
     <meta property="og:image:alt" content="HackUPC landing page screenshot" />
     <!-- <meta property="og:video"       content="https://www.youtube.com/watch?v=qTijleLsync"> -->
     <!-- <meta property="og:video:secure_url" content="https://www.youtube.com/watch?v=qTijleLsync"> -->
 
     <meta name="twitter:card" content="summary_large_image" />
     <meta name="twitter:site" content="@hackupc" />
     <meta name="twitter:creator" content="@hackupc" />
-    <meta name="twitter:title" content="HackUPC 2021 | May 14&ndash;16  | Online hackathon Barcelona" />
-    <meta name="twitter:description" content="Online hackathon in Barcelona on May 14&ndash;16 for 36 horus" />
-    <meta name="twitter:image" content="https://hackupc.com/ogimage.png?v=2021-v1" />
-    <meta name="twitter:image:secure_url" content="https://hackupc.com/ogimage.png?v=2021-v1" />
+    <meta name="twitter:title" content="HackUPC 2022 | April 29&ndash;1 May  | Hybrid hackathon Barcelona" />
```

Comment:
Be carefull! Every where you put 1 May and it's May 1

Reply:
Thanks, that's true